config/bundlesize.js

```diff
@@ -5,15 +5,15 @@ const assetsFolder = 'mobile-wiki/assets';
 module.exports = {
   'mobile-wiki.js': {
     pattern: `${assetsFolder}/mobile-wiki-*.js`,
-    limit: '478KB',
+    limit: '485KB',
```

Comment:
I kind of like that you have to bump these when you change the. kind of...

Reply:
Same.

Reply:
It's just for monitoring if we do not add too much 💩for example while adding some npm packages